package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "arachne-ui",
-  "version": "1.14.57",
+  "version": "1.15.15",
```

Comment:
This version has already been published: https://www.npmjs.com/package/arachne-ui